compiler/damlc/daml-stdlib-src/DA/Ord/Builtin.daml

```diff
@@ -0,0 +1,125 @@
+-- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+{-# LANGUAGE CPP #-}
+
+#ifndef DAML_GENMAP
+module DA.Ord.Builtin where
```

Comment:
Since these are all List functions, how about `DA.List.Fast` or `DA.List.Dedup` or `DA.List.Sort` or something?

Reply:
I think I don’t really like `Fast` in a module name. `Dedup` or `Sort` both work but also only cover half of it. Something like `DA.List.BuiltinOrd` could work. Not sure if that’s better or worse.

Reply:
Ok. I personally don't like `DA.Ord.Builtin` but this isn't important.

Reply:
Do you have another suggestion that somehow combines lists with the fact that the difference is the builtin ordering? Or are you opposed to mentioning the builtin order in the module name at all?

Reply:
How about `DA.List.Order`? As in, both "dedup" and "sort" are imposing some kind of "order" on a list.

Reply:
Not opposed to it but that doesn’t really explain why there is `dedup` in `DA.List` as well. Maybe we can rely on the docs for that but it seemed nice to have that very explicit in the module name.

Reply:
Ok. `DA.List.BuiltinOrder` (or `DA.List.BuiltinOrd`) is fine too. I do think it's better than `DA.Ord.Builtin`.

Reply:
Changed to `DA.List.BuiltinOrder`, agree that it’s better